admin/styles/admin.css

```diff
@@ -237,6 +237,20 @@
 }
 
 @media screen and (max-width: 782px) {
+  #wpwrap {
+    overflow-x: auto;
+  }
+  .h5p-data-view {
+    margin-top: 1em;
+    min-width: 600px; /* width of next media query in hierarchy */
+  }
+  .h5p-data-view thead th {
+    vertical-align: top;
+    word-wrap: break-word;
+  }
+  .h5p-data-view tbody td, .h5p-data-view tfoot td {
+    display: table-cell!important; /* WP media query has priority */
```

Comment:
Please add a space

Reply:
@fnoks In front of _!important_? I guess so ...

Reply:
@fnoks There you go ...